ctapipe/reco/shower_processor.py

```diff
@@ -25,11 +25,14 @@ class ShowerProcessor(Component):
     Input events must already contain dl1 parameters.
     """
 
-    reconstructor_type = create_class_enum_trait(
-        Reconstructor,
-        default_value="HillasReconstructor",
-        help="The stereo geometry reconstructor to be used",
-    )
+    reconstructor_types = List(
```

Comment:
The help message now now longer shows the list of possible reconstructor types (a known problem when we change from an enum  to a list of enums...).  It would be good to explicitly add the class list to the `help=` option as we did elsewhere. 

Here is what we get now:
```
ShowerProcessor(Component) options
----------------------------------
--ShowerProcessor.reconstructor_types=<list-item-1>...
    The stereo geometry reconstructors to be used
    Default: ['HillasReconstructor']
```

And this is what was there before (note the list of names):
```
--ShowerProcessor.reconstructor_type=<CaselessStrEnum>
    The stereo geometry reconstructor to be used
    Choices: any of ['HillasReconstructor', 'HillasIntersection'] (case-insensitive)
    Default: 'HillasReconstructor'
```

Could just add this to the help string:
```py3
f"Choices are: {list(Reconstructor.non_abstract_subclasses().keys())}" 
```

Reply:
Done